justfile

```diff
@@ -23,6 +23,19 @@ features := '\
     --features=default \
     --no-default-features \
     '
+check:
```

Comment:
There should probably be a empty line above this line.

Reply:
Didn't see that close quote.  Fixed.